server/data/projects.js

```diff
@@ -93,6 +93,23 @@ const createProject = async (projectObjParam, user) => {
 	);
 	return createdProject;
 };
+const likeProject = async (user, project) => {
+	const userInfo = user;
+	const userId = userInfo._id;
+	const projectId = isValidObjectId(project);
+	isValidObjectId(userId);
+	const projectCollection = await projects();
+	const getProjectInfo = await getProjectById(projectId);
+	if (!getProjectInfo) {
+		throw internalServerErr('Project could not be found');
```

Comment:
getProjectById will throw a not found error if the project is not found, you don't need to do this here